jetbrains-core/tst/software/aws/toolkits/jetbrains/services/lambda/python/PythonLambdaHandlerResolverTest.kt

```diff
@@ -140,6 +141,21 @@ class PythonLambdaHandlerResolverTest {
         assertHandler("app.handle", true)
     }
 
+    @Test
+    fun pyTestHandlersIgnored() {
+        createPyTestHandler("hello_world/app.py")
+
+        assertHandler("hello_world/app.handle", false)
```

Comment:
This will always be false because the method name is test_handle

Reply:
Blew up all of these new tests and rewrote them.